scripts/34-Plain-Text-Team.md

```diff
@@ -0,0 +1,601 @@
+
+Here we've got a heading, denoted by the hash symbol, italic with underscores, bold with double asterisks, and links using this bracket pairing syntax.
+
+There are a few more options available, which you can look up at markdownguide.org, but this is the overwhelming majority of formatting you'll need on a day-to-day basis.
+
+---
+
+![[obsidian-kanban-paint.png]]
+
+notes:
+
+The genius of storing your data in this universal plain-text format is that should you wish to migrate from GitHub to another similar platform, your data is portable and under your control.
+
+GitHub formats Markdown very nicely, but you can export it in any format you like, and edit it with any tool you like, present and future.
+
+Including my favourite tool here, Obsidian.
+
+Markdown keeps your team focussed on what is important by allowing you just enough formatting, but no rich customisation options.
+You're not making a beautiful client brochure, so you shouldn't use 90s desktop publishing tools to make your company's critical documentation.
+Should you?
+
+Back to GitHub's features:
+
+---
+
+![[issues.png]]
+
+## Issues
+
+notes:
+
+Though not part of the git system, Issues are a natural addition that all git hosts have implemented:
+A simple task system for capturing work that needs doing.
+These could be new features, customer requests, bug reports, or ideas.
+They have a rich comment thread for discussion, can be assigned to team members, and tagged with custom tags.
+
+This minimum viable project management system could be all you need.
+Certainly, for a solo or small team, capturing requirements in Issues might be enough.
+
+But if you need more, you need Milestones.
+
+---
+![[milestones.png]]
+
+## Milestones
+
+notes:
+GH milestones are a grouping of issues with a deadline.
+
+They typically represent a target, a release, or something the team is working towards.
+Milestones just have a title, a description, and a date.
+No burndown charts, no swim lanes, no complex statistics, just a progress bar.
+
+This might be enough project management for you.
+If not, it is time for GH Projects.
+
+---
+![[projects.png]]
+
+## Projects
+
+notes:
+
+Each GitHub repo, team, and organisation can have a project board, a lightweight kanban board with customisable columns, allowing you to group your issues together, and observe their progress through your current iteration.
+
+This represents the information radiator for your team, a bird's-eye view of what is happening with the project, and something you might gather around for your morning catchup meeting.
+
+You don't need all the features of Trello, JIRA, or anything like that. You need the minimum viable board.
+
+This bare-bones tool completely side-steps 'the JIRA effect', which is if you have a tool that is packed full of time tracking, velocity points, and so on, the temptation is to use all these features, even if they give no value, and complicate your processes.
```

Comment:
I do like the simplicity of GitHub projects. But as far as I can tell it's not actually a plaintext format is it? It's a proprietary, locked-in tool that you happen to believe is better than Jira partly because it's better designed but also because it's integrated into GitHub, a benefit which relies on GitHub's monopoly position. Am I wrong?

Reply:
true, same as issues. Neither are plain text, but both are the best tool for working in a plain-text team, based on gh.